### DISCUSSION TITLE
Happy Hacktober 🎃  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#35357] [New to GitHub]
The most known holiday in October may be Halloween, but in the open source world, the biggest event is Hacktoberfest! 

What is [Hacktoberfest](https://hacktoberfest.com/)? Digitalocean’s annual event for nine years running that celebrates open source contributions, giving back to projects that don’t have the resources to expand, and expand your skills ✏️. There are ways to get involved even with low/non-code contributions as well. The perfect way to jumpstart your GitHub journey is by participating in this much-loved month-long event. Check out the details below ⬇️. 

🌟**Community Superstars**: A round of applause for these community citizens 👏
* @WabashFan for not only giving tons of background information in their [question](https://github.com/orgs/community/discussions/34734), but coming back to self-solve it as well - nice ​​! 
* @welkeyever: Asked a great [question](https://github.com/orgs/community/discussions/34687) on the dependency graph and included THREE examples - help them out by answering! 
* @sailingthroughemotion: is new to GitHub and Pages, but they’re already trying out some new themes! Kudos to you and if anyone can help out on [why their theme disappeared](https://github.com/orgs/community/discussions/34359) - dive in 🤿.

👻 **‘Tis the Season**: Hacktoberfest has started and we’ve got you covered 💻
* **How to Get Ready**: @mishmanners’s [guide](https://dev.to/github/how-to-get-ready-for-hacktoberfest-2022-2ck2) to getting started in Hacktoberfest 2022 ✅! 
* **Expand your open source contributions**: This year, we’re encouraging more than code contributions: writing, design, advocacy, and financial donations 💖. [Learn more](https://github.blog/2022-10-04-expand-your-open-source-contributions-during-hacktoberfest-2022/) from @abbycabs. 
* **Hacktoberfest Repositories**: Looking to get on the action? @OtacilioN has you covered with their curated [list](https://github.com/OtacilioN/awesome-hacktoberfest-2022) of Hacktoberfest 2022 repositories. There is even a section of repositories great for beginners!

🆕 **What’s new in GitHub**: Around the product and the Community, the latest. 
* **Private Profiles** 🔒: An oft-requested feature, private profiles are now [generally available](https://github.blog/changelog/2022-09-29-private-profiles-are-now-generally-available/). Learn how to set your profile to private [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/setting-your-profile-to-private). 
* 🗳️ **Introducing the Repositories Category**: Repositories now has its own [category](https://github.com/orgs/community/discussions/categories/repositories) in Community on Discussions, use it to ask questions, share feedback, and collaborate on all things GitHub repository-related. 
* **Local Time Zones Available on Profiles** 🕰️: Profile personalization continues with this latest [rollout](https://github.blog/changelog/2022-09-23-local-timezones-available-on-profiles/). 
* **GitHub Skill**🆙: Expand your GitHub skillset each week with @ossanna16, [this week](https://github.com/orgs/community/discussions/35059) create your profile README 📖. 

How are you celebrating Hacktoberfest, the changing of the seasons, or just that you’ve gotten started with GitHub? We want to know in the comments.

**Comment by OtacilioN** [comment]
Heeeey @queenofcorgis thanks for this amazing discussion post. For those who are going to participate in the Hacktoberfest, welcome, and have a happy hacking

**Reply by queenofcorgis** [reply]
@OtacilioN thanks for compiling such an awesome resource 💥 Happy Hacktober 👻 !

**Reply by Lelandj88** [reply]
I am new to all of this ( as you will learn by my obvious ignorance to any/all of this websites content) and I'm trying to understand what exactly I keep finding on my phone that inevitably always leads me back to this website.
 My wife and I split up exactly a year ago this week, about a week or so after I moved out I noticed that my phone/Google account had been hacked and I was being monitored in someway shape or form. Again, I'm not necessarily"tech-savvy" nor do I have an interest in coding or computers in general. I'm a blue collar kind of guy who has a passion for working with my hands and getting dirty while I'm doing it. With that being said, I have nothing against anyone who enjoys doing the kind of work that is on GitHub. 
 So after I noticed that I had been hacked I did all the normal stuff... New password, new account new phone ect... But as soon as I would log in with my name or recover my account, any new device I used was immediately taken over... At first it was just basically letting me know that they were there and that there was nothing I could do about it... And unfortunately they were right. But I started downloading my data from my Google account and figured out how to open the ZIP files that I was extracting. And two things were always at the end of the rabbit holes I went down, the apache 2.0 license and this website.... I don't know much but if anyone would like to help me better understand what these things are and how I could possibly defeat my hacker and ultimately find out who they are it would be greatly appreciated thank you for taking the time to read this and I hope to hear from someone soon

**Reply by 3kh0** [reply]
@Lelandj88 Woah, that is quite the experience! GitHub does host malware, although for educational purposes. It does not talk long to find it, I would suggest you switch to iPhones since they have better protection from malware and such. Get a 12 or 13, (unless you really want that *dynamic island*). Get someone to check your router, it seems like a router or a modem issue. I do not know much in this area, good luck to you!

**Reply by evans2101** [reply]
TEST

**Reply by BlackShackle** [reply]
Well I should have been here long ago but I kind of stopped using my computer. While I am not a hacker by any means, my ability to even protect myself has disappeared now days. It's difficult to go without ever sleeping so that my gear stays the way I put it. I do appreciate the help, and I hope that all of us find the answers we seek. And my piece of info for the discussion is that the iPhone is not going to help. Lol. I probably don't know how to use it though. I have had android for so long and when I bought one to try and get out of this exact circumstance, I found I was unable to leverage the power that Apple does offer. To ask a question, Does anyone know how I might go about forcing a hard reset of my android phone if the buttons are not responding? Also how I might get my phone calls to stop forwarding to a call center in Davo? All help would be greatly appreciated.

**Reply by katakume21** [reply]
lol

**Reply by shumalov0** [reply]
rez

**Reply by SK-Luffa** [reply]
lue lue lue

**Comment by 3kh0** [comment]
Great post summing things up for us @queenofcorgis! Just wanted to ask if raising the GitHub push limit is on the GitHub roadmap. Projects are getting bigger and big so why should the limit stay the same? Unity files can easily exceed 100MB and yes I know Git LFS is an option but it is a pain in the neck to use and the limits are small, I exceeded the limit within 5 days of me uploading a file. Thank you again for the post!

**Reply by queenofcorgis** [reply]
Hi there! Glad you enjoyed the post, thanks for the kind words 😄 . Have you made a discussion about this request here? I don't see one, but once you do feel free to link me or tag me so I can get the appropriate team on visibility 👀

**Reply by 3kh0** [reply]
@queenofcorgis No problem! There was already a discussion, not directly suggesting the feature but still there. #68

**Comment by muhirwarobert** [comment]
Hi

**Reply by muhirwarobert** [reply]
0789052246

**Comment by Christhino** [comment]
Hey

**Comment by ghost** [comment]
18644201478",

**Comment by SamNickGammer** [comment]
dzfgf

**Comment by Jaures-beinjamin** [comment]
thank you for the article

**Comment by Jaures-beinjamin** [comment]
no problem i will thank you

**Comment by Jaures-beinjamin** [comment]
I have a question how can I be part of the github staff like you?

**Reply by 3kh0** [reply]
get a job

**Comment by Preetiraj3697** [comment]
thank you for the this discussion

**Comment by Jaures-beinjamin** [comment]
I have a question how can I be part of the github staff like you?

Jaures Beinjamin FOTSING
INFORMATICIEN  | DÉVELOPPEUR


Email: ***@***.***
Github: https://github.com/beinjamin
LinkedIn: https://www.linkedin.com/in/jaures-beinjamin-fotsing-297069211

**Reply by 3kh0** [reply]
get a job

**Comment by welkeyever** [comment]
I've searched some similar cases according to my [question](https://github.com/orgs/community/discussions/34687), which ends up a "run a job to redetect" to solve. Seems like an internal operation from Github? 
like these: 
* https://github.com/orgs/community/discussions/23565
* https://github.com/orgs/community/discussions/21637

Am I asking this question in the wrong place? If there is a correct way to raise this kind of question, please let me know~ Thanks!

**Reply by queenofcorgis** [reply]
Thanks for raising this @welkeyever - I've reached out to some folks to see if I can find some answers 🔍 .

**Reply by sabrican3532** [reply]
Facebook hesabım çalındı lutven geri almak istiyorum çaldılar yardım ci olun lutven benim adimca ciplak resimler gunderiyor lutven

**Comment by sabrican3532** [comment]
https://www.facebook.com/sabri.can.3532

**Comment by Veleniss** [comment]
Research and learn

**Comment by melvincwng** [comment]
Happy Hacktoberfest 2022 everyone :D 🎃🎃🎃

**Reply by 3kh0** [reply]
To you too!

**Reply by Preetiraj3697** [reply]
To you too

**Comment by alaa19921** [comment]
لماذا لاتضهر كلمات السر لشبكات الواي فاي

**Reply by Ahmedgkanria** [reply]
هبقي اظهرها

**Comment by Veleniss** [comment]
In regards to the  forwarding it's pretty simple you you have to go to your
phone settings and disable your call forwarding option.
To ask you a  question, did you figure out the solution to the question
below because I'm having the same issues on my android device.

Does anyone know how I might go about forcing a hard reset of my android
phone if the buttons are not responding

**Comment by oswoc** [comment]
Happy Hacktoberfest!!, may you achieve your goal happy hacks! 🙌🏻

**Reply by PhillipMike** [reply]
Hey can you help me srt this up

**Reply by oswoc** [reply]
Hi!  @PhillipMike :)
What's up bro? How are you?

**Reply by PhillipMike** [reply]
Wassup

**Reply by 3kh0** [reply]
I just finished and I am going to get the tshirt

**Reply by durgesh2001** [reply]
Happy october, hope you all are doing well. Hacktoberfest is just about to end.Keep contributing and keep learning.

**Reply by preciousebuka92gmailcom** [reply]
Hi

**Comment by mariadhari6** [comment]
hello

**Comment by PhillipMike** [comment]
Can u help me set up readme

**Reply by oswoc** [reply]
Yeah @PhillipMike 

> first create a repo, on the name of repo add your name(username)
> Then you will be created a file readme.md or if you do not create a file with that extension readme.md and that's it, you would have your card of details customizable :)

**Reply by PhillipMike** [reply]
No I didn't but I need one I always forget my info

**Reply by oswoc** [reply]
I understand you bro, in the Readme.md is very customizable how for example, https://github.com/anuraghazra/github-readme-stats In this repository there are several ideas on how to customize the Readme, add languages, details of various things, It's cool. I recommend it 👌🏻

**Reply by Preetiraj3697** [reply]
Can I help you to build readme like me https://github.com/Preetiraj3697

**Comment by PhillipMike** [comment]
Car

**Comment by uesleibros** [comment]
Hi!

**Comment by Jaures-beinjamin** [comment]
Yes I'm well and you









Jaures Beinjamin  ✅️

Software Engineering |
IT Orientation Coach |
IT Recruiter |

Email: ***@***.***
Github: https://github.com/beinjamin
LinkedIn: https://www.linkedin.com/in/jaures-beinjamin-fotsing-297069211

Le mar. 25 oct. 2022, 00:51, wangbang1128 ***@***.***> a
écrit :

**Comment by sabrican3532** [comment]
@sabrican3532 cod
